assets/helpers/contributions.js

```diff
@@ -51,9 +51,9 @@ export const logInvalidCombination = (contributionType: ContributionType, paymen
 };
 
 export type ThirdPartyPaymentLibraries = {
-  ONE_OFF: { Stripe: Object },
-  MONTHLY: { Stripe: Object, PayPal: Object },
-  ANNUAL: { Stripe: Object, PayPal: Object },
+  ONE_OFF: { Stripe: Object | null },
```

Comment:
Could you use [?Object](https://flow.org/en/docs/types/maybe/) here?

Reply:
I prefer the `<type> | null` pattern, as it's more specific. `?<type>` can be `<type>`, `null` or `undefined`